Branch-SDK/Branch-SDK/BNCCrashlyticsWrapper.m

```diff
@@ -0,0 +1,71 @@
+//
+//  BNCCrashlyticsReportingHelper.m
+//  Branch.framework
+//
+//  Created by Jimmy Dee on 7/18/17.
+//  Copyright © 2017 Branch Metrics. All rights reserved.
+//
+
+#import "BNCCrashlyticsWrapper.h"
+
+@interface BNCCrashlyticsWrapper()
+@property (nonatomic, nullable) id crashlytics;
+@end
+
+@implementation BNCCrashlyticsWrapper
+
++ (id)sharedInstance
```

Comment:
I'll check to see what Google style and Apple style mean and try to follow the former in the future. I agree legibility should be the main goal with code style.

I've been working a lot lately with linting tools like `eslint` and `rubocop`. I get a little annoyed when they want to change my style, but the fact that you can run `eslint --fix` or `rubocop -a` to fix problems automatically makes it painless. With automation, you can have a strict style and less friction. Are there any standard Obj-C linters out there? I use Structure > Reindent all the time in Xcode. That doesn't change the location of brackets, but it makes my disagreements with Xcode painless, and it's too useful to fight with.